versioned_docs/version-2.0-2.4/reference-guides/cluster-configuration/rancher-server-configuration/rke1-cluster-configuration.md

```diff
@@ -8,7 +8,7 @@ This section covers the configuration options that are available in Rancher for
 
 You can configure the Kubernetes options one of two ways:
 
-- [Rancher UI](#rancher-ui-options): Use the Rancher UI to select options that are commonly customized when setting up a Kubernetes cluster.
+- [Rancher UI](#configuration-options-in-the-rancher-ui): Use the Rancher UI to select options that are commonly customized when setting up a Kubernetes cluster.
```

Comment:
This link doesn't seem to go anywhere? Trying #rancher-ui-options leads to the correct in-page header